exporter-api/src/main/java/io/zeebe/exporter/api/context/Controller.java

```diff
@@ -34,4 +34,6 @@
    * @param task the task to run
    */
   void scheduleTask(Duration delay, Runnable task);
+
+  ScheduledTask scheduleCancellableTask(final Duration delay, final Runnable task);
```

Comment:
Do we need a new method for this? Or just change the return type of `scheduleTask`?

Reply:
I'm thinking about backward compatibility in this case because I'm thinking that this is a part of public API or I'm wrong? :)

Reply:
That's right. It is a public API. Changing return type from void to a non-void object will not break source compatibility - but it will beak binary compatibility. @npepinpe I guess then it will be an issue for external exporters? So we have to keep two different methods with void and non-void return types.

Reply:
Keep in mind that we don't expect many exporter authors to be implementing their own controller. At most, they might be mocking it. I would still mark the void method as deprecated however so already they start using the new method.

Reply:
@deepthidevaki Done, I've added a deprecation JavaDoc and annotation